src/odata-to-abstract-sql.ts

```diff
@@ -1275,6 +1280,22 @@ export class OData2AbstractSQL {
 		}
 		throw new SyntaxError('Could not match operand');
 	}
+	EqualsAny(match: any): EqualsAnyNode | undefined {
+		if (match[0] !== 'eqany') {
+			return;
+		}
+
+		if (match[1][0] !== 'Bind') {
+			throw new SyntaxError(
+				'Equals any must have a bind as the second argument',
+			);
+		}
+		return [
+			'EqualsAny',
+			this.Operand(match[0]),
+			this.Operand(match[1]) as BindNode,
```

Comment:
How about making an `isBind` thing in abstract-sql-compliler like we have for the rest?